src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Header = () => (
+  <nav className="navbar">
+    <h1 className="navbar-brand">Bookstore CMS</h1>
+    <ul className="navbar-nav">
+      <li className="nav-item">
+        <NavLink to="/" activeClassName="active" exact>Books</NavLink>
+      </li>
+      <li className="nav-item">
+        <NavLink to="/categories" activeClassName="active" exact>Categories</NavLink>
```

Comment:
- [ ] Please use the `<Link>` component instead of the `<Navlink>` one to comply with the project requirements for these 2 elements.

Reply:
As stated on React Router's official website, [NavLink](https://reactrouter.com/web/api/NavLink) is a special version of the `<Link>` that will add styling attributes to the rendered element when it matches the current URL. So this required change was already fulfilled.